### ISSUE TITLE
[Timeline][A11y]: Focus indicator is cut off for the timeline item

### ISSUE THREAD

**Issue by Brahmansh** [SAP/ui5-webcomponents#11386]
### Bug Description

Focus indicator for the timeline item is cut off when focused.

### Affected Component

Timeline

### Expected Behaviour

Focus indicator should be visible without getting cut off.

### Isolated Example

https://sap.github.io/ui5-webcomponents/components/fiori/Timeline/

### Steps to Reproduce

1. Go to this link - https://sap.github.io/ui5-webcomponents/components/fiori/Timeline/
2. Press tab or click on any timeline item inside basic sample window.
3. Observe focus indicator is cut off.

### Priority

None

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Organization

SFSF

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by TeodorTaushanov** [comment]
Hello @SAP/ui5-webcomponents-topic-b, can you check this issue?
